pyproject.toml

```diff
@@ -44,7 +43,7 @@ dependencies = [
 # These optional dependencies are for building the
 # application as a part of a Github Action.
 ci = [
-    'pyinstaller==6.5.*',
+    'pyinstaller==6.10.*',
```

Comment:
Does 6.10 work for the application on all platforms? Qt5 might be a problem with new pyinstallers so let's make sure.

Reply:
Confirmed that the produced artifacts run on my windows machine and on Nico's osx on apple silicon